examples/hikari_lightbulb_basic_queue/music_plugin.py

```diff
@@ -1,13 +1,14 @@
-import os
 import logging
-
-from consts import TOKEN, PREFIX
+from typing import Any
+from typing import Optional
```

Comment:
These can be merged or just import typing and reference them as `typing.x`

Reply:
in the pure hikari example it is done like this (so I thought I'd stick to that) and the isort config hikari uses forces a single line for every import

Reply:
vicky, what should i do?

Reply:
https://5124.16-b.it/ss/12:23:55_01-10-2021.png
pep8 states that single line, multi imports with from are fine, and it's cleaner that way, so it should be that way.
Do this for the other 2 reviews stating the same.

Reply:
Ok, I'll prolly just have to change the isort config